### ISSUE TITLE
Don't work at all

### ISSUE THREAD

**Issue by BirkhoffLee** [willshiao/node-bash-obfuscate#1]
```
birkhoff at Birkhoff-MBPR in ~/dev/tesasd
$ bash-obfuscate
env: node\r: No such file or directory
```

```
$ bash-obfuscate 123.sh -o 321.sh
env: node\r: No such file or directory
```

**Comment by willshiao** [comment]
This was caused by Windows-style CRLF line endings in `bin/cli.js`. Funnily enough, I only tested this on a Windows computer (and ran the output of the script through a bash shell).

I fixed this in 83822be (v1.1.1) by normalizing line endings of `.js` files.

**Comment by BirkhoffLee** [comment]
Thanks for the fast reply. I'll test later today and see if it works.
